extensions/file-manager/package.json

```diff
@@ -71,21 +71,25 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.25.4",
-    "filesize": "^8.0.6"
+    "@raycast/api": "^1.46.1",
```

Comment:
Since I upgraded this Raycast lib, I wanted to update the deprecated Raycast APIs to the new Raycast APIs. 
Please advise me if I need to exclude this change to a separated PR.